bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -160,7 +160,12 @@ case object RemoteBackend extends CachedBackend {
 
     try {
       client.executeMethod(method)
-      VoTableParser.parse(e.url, method.getResponseBodyAsStream).fold(p => QueryResult(e.query, CatalogQueryResult(TargetsTable.Zero, List(p))), y => QueryResult(e.query, CatalogQueryResult(y).filter(e.query)))
+      VoTableParser.parse(e.url, method.getResponseBodyAsStream).fold(p => QueryResult(e.query, CatalogQueryResult(TargetsTable.Zero, List(p))), y => {
+        // Print selected stars for debugging
+        // TODO Remove
+        CatalogQueryResult(y).targets.rows.foreach(println)
```

Comment:
Yes, Andy wants to see the output of parsing the VoTable xml which isn't really human-parsable